source/getting-started/admin-onboarding-tasks.rst

```diff
@@ -13,146 +13,161 @@ Administrator Onboarding Tasks
   
-**Migrating to AD/LDAP or SAML from email-based authentication**
+Migrate to AD/LDAP or SAML from email-based authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- If you have Professional or Enterprise plans, you can migrate from email authentication to Active Directory/LDAP or to SAML Single Sign-on. To set up Active Directory/LDAP, see `Active Directory/LDAP Setup <https://docs.mattermost.com/onboard/ad-ldap.html#active-directory-ldap-setup-e10-e20>`_. To set up SAML Single Sign-on, see `SAML Single-Sign-On <https://docs.mattermost.com/onboard/sso-saml.html>`_.
+- Mattermost Professional or Enterprise customers can migrate from email authentication to Active Directory/LDAP or to SAML Single Sign-on. To set up Active Directory/LDAP, see `Active Directory/LDAP Setup <https://docs.mattermost.com/onboard/ad-ldap.html#active-directory-ldap-setup-e10-e20>`_. To set up SAML Single Sign-on, see `SAML Single-Sign-On <https://docs.mattermost.com/onboard/sso-saml.html>`_.
 - After the new authentication method is enabled, existing users cannot use the new method until they go to **Settings > Security > Sign-in method** and select **Switch to using AD/LDAP** or **Switch to using SAML Single Sign-on**. After they have switched, they can no longer use their email and password to sign in.  
   
-**Deactivating a user**
+Deactivate a user
+~~~~~~~~~~~~~~~~~
+
+System Admins can go to **System Console > Users** for a list of all users on the server. Search and filter the list to make finding users easier. Select the user's role, then choose **Deactivate**. To preserve audit history, users are typically never deleted from the system. 
 
-- System Admins can go to **System Console > Users** for a list of all users on the server. The list can be searched and filtered to make finding the user easier. Click the user's role and in the menu that opens, click **Deactivate**.
-- To preserve audit history, users are typically never deleted from the system. If permanently deleting a user is necessary (e.g. for the purposes of `GDPR <https://gdpr-info.eu/>`__), an `mmctl command <https://docs.mattermost.com/manage/mmctl-command-line-tool.html>`__ or a `CLI command <https://docs.mattermost.com/manage/command-line-tools.html>`_ can be used to do so.
-- Note that AD/LDAP user accounts cannot be deactivated from Mattermost; they must be deactivated from your Active Directory.
+If permanently deleting a user is necessary (e.g. for the purposes of `GDPR <https://gdpr-info.eu/>`__), an `mmctl command <https://docs.mattermost.com/manage/mmctl-command-line-tool.html>`__ or a `CLI command <https://docs.mattermost.com/manage/command-line-tools.html>`_ can be used to do so. AD/LDAP user accounts can't be deactivated from Mattermost; they must be deactivated from your Active Directory.
 
-**Checking for a valid license in Enterprise Edition without logging in**
+Check for a valid license in Enterprise Edition without logging in
```

Comment:
Should this be "an enterprise edition"?

Reply:
Good question! I wonder if this information applies equally to Professional as it does to Enterprise. If so, removing the plan details from the title and moving it to the description would work well.

Reply:
@cwarnermm, following up on this. :)